simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -2,22 +2,24 @@
 <sdf version="1.5">
   <model name="wamv">
     <link name="base_link">
-      <!-- TODO: Tune these -->
+      <gravity>0</gravity>
```

Comment:
I think you're essentially making the boat a massless object in 0g so you can just set it's state, but is that necessary?